dashboard/src/components/model/ModuleParameters.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <div class="module-parameters">
+    <b>Parameter source:</b>
+    <select v-model="parameterSource" name="module-parameter-source">
+      <option>User Supplied</option>
+      <option>Browse Database</option>
+    </select>
+    <br />
+    <!-- If user selects something other than User Supplied, display the
+         list of inverters from the db. This should probably be it's own
+         component with some search functionality built in
+     -->
+    <div v-if="parameterSource !== 'User Supplied'">
+      <b>Select a Module:</b>
+      <select @change="loadModule" name="module-list">
+        <option v-for="p in parameterOptions" :key="p">{{ p }}</option>
+      </select>
+      <br />
+    </div>
+    <div v-if="model == 'pvsyst'">
+      <b>gamma_ref:</b>
+      <input type="number" v-model.number="parameters.gamma_ref" />
+      <help :helpText="this.definitions.properties.gamma_ref.description" />
+      <br />
+      <span style="color:#F00;" v-if="'gamma_ref' in this.errors">
+        {{ this.errors.gamma_ref }}
+        <br />
+      </span>
+      <b>mu_gamma:</b>
+      <input v-model="parameters.mu_gamma" />
+      <help :helpText="this.definitions.properties.mu_gamma.description" />
+      <br />
+      <span style="color:#F00;" v-if="'mu_gamma' in this.errors">
+        {{ this.errors.mu_gamma }}
+        <br />
+      </span>
```

Comment:
Why does gamma_ref have `type="number" v-model.number=` but mu_gamma doesn't?

Reply:
maybe this is causing one validation issue (along with the added example in the openapi spec)

![image](https://user-images.githubusercontent.com/4625457/101401779-0a0cf700-3890-11eb-82b2-c3db14ad4e7b.png)